Feature/event columns sorting (#2wgk5f)

jasynecka opened the pull request.

jasynecka committed:
add initial sorting function, add data attributes to colomn headers
#2wgk5f

jasynecka committed:
add sorting order
#2wgk5f

jasynecka committed:
add pointing triangle, add mixins and css styles, code refactoring
#2wgk5f

jasynecka committed:
add feature to readme file
#2wgk5f

jasynecka committed:
merge from develop, solve conflict in readme
#2wgk5f

jasynecka committed:
add assets/dist to .gitignore
#2wgk5f

jasynecka committed:
preserve sorting after event added/paused/unpaused
#2wgk5f

jasynecka committed:
Code refactoring
#2wgk5f

jasynecka committed:
add sorting events table when window reloaded
#2wgk5f

jasynecka committed:
clean preserved sorting when window reloaded, code refactoring
#2wgk5f

jakubmikita commented on assets/src/js/event-sorting.js:
```diff
@@ -0,0 +1,98 @@
+
+	$( '.tools_page_advanced-cron-manager' ).on(
+		'click',
+		'#events .header .event, #events .header .schedule, #events .header .next-execution',
```
Wouldn't be better to introduce `.is-sortable` class?

jakubmikita commented on assets/src/js/event-sorting.js:
```diff
@@ -0,0 +1,98 @@
+		return 0;
+	}
+
+	// remove preserved events sorting when window is reloaded.
```
You shouldn't clear it I think. It's a desired side effect that the sorting stays as it was

jakubmikita commented on views/parts/events/row.php:
```diff
@@ -61,7 +61,7 @@
 			</div>
 		</div>
-		<div class="column schedule"><?php echo esc_html( $schedules->get_schedule( $event->schedule )->label ); ?></div>
+		<div class="column schedule" data-interval="<?php echo esc_attr( $event->interval ); ?>"><?php echo esc_html( $schedules->get_schedule( $event->schedule )->label ); ?></div>
```
I like that you used an interval here instead of the label 👍

jasynecka committed:
select sortable columns by special class
#2wgk5f

jasynecka committed:
set ascending default sorting for column next-execution
#2wgk5f

jasynecka committed:
apply preserved sorting when window is reloaded
#2wgk5f

jasynecka committed:
solve bug with zero interval after event paused
#2wgk5f

jasynecka committed:
change impementation to wp hooks in js
#2wgk5f

jasynecka committed:
synchronize events table footer with header when sorting
#2wgk5f

jasynecka committed:
fix bug when filtered events are not sorted correctly
#2wgk5f

jakubmikita committed:
change chevron behaviour

#2wgk5f remove missleading hover state and
make only sortable column have pointer cursor
